src/App.tsx

```diff
@@ -15,48 +22,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  isReversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((goodA, goodB) => {
+      switch (sortField) {
+        case SortType.Alphabet:
+          return goodA.localeCompare(goodB);
+
+        case SortType.Length:
+          return goodA.length - goodB.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortFiled, setSortFiled] = useState<SortType>(SortType.Empty);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const visibleGoods = getPreparedGoods(
+    goodsFromServer,
+    sortFiled,
+    isReversed,
+  );
+
+  const reset = sortFiled || isReversed;
+
+  const handleReset = () => {
+    setSortFiled(SortType.Empty);
+    setIsReversed(false);
+  };
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          className={cn(
+            'button',
+            'is-info',
```

Comment:
```suggestion
            'button is-info',
```

can be more simplify